Envfile

```diff
@@ -163,6 +163,10 @@ variable :TWITCH_WEBHOOK_SECRET, :String, default: "Optional"
 # (https://api.stackexchange.com/docs)
 variable :STACK_EXCHANGE_APP_KEY, :String, default: ""
 
+# For setting the IDs of the staff and welcoming users
+variable :STAFF_USER_ID, :Integer, default: 1
+variable :WELCOMING_USER_ID, :Integer, default: 1
```

Comment:
Heh, not sure how long this will be around. I suspect once @benhalpern addresses #6049, we will want the welcoming user account to be set the same way for consistency's sake 😆